ecologits/tracers/utils.py

```diff
@@ -1,10 +1,13 @@
+import logging
 from typing import Optional
 
 from ecologits.electricity_mix_repository import electricity_mixes
 from ecologits.impacts.llm import compute_llm_impacts
 from ecologits.impacts.modeling import Impacts, Range
 from ecologits.model_repository import models
 
+logger = logging.getLogger(__name__)
```

Comment:
I would define a logger in a log.py and import it wherever we need logging.

```python
import logging

logger = logging.getLogger("ecologits")
```

Reply:
Isn't it exactly the same? I thought the logger was determined by the name only

Reply:
Yes, indeed from a practical standpoint it does not change anything, but I see 3 advantages :

1. Do not display library submodule in logs

If we define the logger with `logging.getLogger(__name__)` in the root of the package or in a submodule, we will get difference logging sources.

```
[DATE] [LEVEL] ecologits: message
[DATE] [LEVEL] ecologits.submodule: message
```

I think it is preferred to only log with "ecologits" unless if it makes sense to have submodules. In our case, the end user only sees the EcoLogits object and should not know about tracers, for instance. Plus, it makes filtering easier for the end user, just target "ecologits" and that's it.

2. It removes the ambiguity or possibility of errors

Removes the possibility of having a typo in the name of the logger, for instance. (though it shouldn't happen with good reviewing, we can eliminate the risk by configuring it once).

3. If we define some customizations on the logger we only do it once.

For instance, if we want to automatically deduplicate messages, we could add a specific function for that. Once it is implemented in the logger, we no longer need to care whether it is available or not in other parts of the library.